go/validate.go

```diff
@@ -214,10 +228,9 @@ func parseChoices(choiceStrings []string) []Choice {
 }
 
 type DropdownField struct {
-	Type          string        `json:"type"` // "Dropdown"
-	Choices       []string      `json:"choices"`
-	Filter        *ChoiceFilter `json:"filter,omitempty"`
-	parsedChoices []Choice
```

Comment:
no need to store it, we'll parse out the []Choice from []string at the time when we validate